spec/Organization.tex

```diff
@@ -137,17 +132,15 @@
 
 \item
 Chapter~\ref{Standard_Modules}, Standard Modules, describes the
-standard modules that are provided with the Chapel language.
+structure of standard modules that are provided with the Chapel language
+and the \chpl{Base} standard module.
```

Comment:
Just happened to stumble across this:  It looks to me like much of what's in Standard_Modules.tex ought to be documented as part of the current internal module chpldoc push -- shall I send out a list of routines that it documents and see whether we've got them covered?  (e.g., I know I've done Assert by now).